src/pay/utils/getPayErrorCode.ts

```diff
@@ -0,0 +1,22 @@
+import {
+  GENERAL_PAY_ERROR_CODE,
+  PAY_INVALID_CHARGE_ERROR_CODE,
+  PAY_INVALID_PARAMETER_ERROR_CODE,
+  PAY_TOO_MANY_REQUESTS_ERROR_CODE,
+} from '../constants';
+
+export function getPayErrorCode(errorCode?: number) {
```

Comment:
So it seems we are generating short Error Messages, so let's rename this as `getPayErrorMessage`.

Reply:
Makes sense to me, renamed this function and constants.